contracts/minter/src/contract_tests.rs

```diff
@@ -638,33 +653,49 @@ fn check_per_address_limit() {
         buyer.clone(),
         minter_addr.clone(),
         &mint_msg,
-        &coins(PRICE, NATIVE_DENOM),
+        &coins(UNIT_PRICE, NATIVE_DENOM),
     );
 
     assert!(res.is_ok());
 
     // second mint fails from exceeding per address limit
     let mint_msg = ExecuteMsg::Mint {};
-    let res = router.execute_contract(buyer, minter_addr, &mint_msg, &coins(PRICE, NATIVE_DENOM));
+    let res = router.execute_contract(
+        buyer,
+        minter_addr,
+        &mint_msg,
+        &coins(UNIT_PRICE, NATIVE_DENOM),
+    );
     assert!(res.is_err());
 }
 
 #[test]
 fn batch_mint_limit_access_max_sold_out() {
     let mut router = custom_mock_app();
     let (creator, buyer) = setup_accounts(&mut router).unwrap();
+
+    let creator_balances = router.wrap().query_all_balances(creator.clone()).unwrap();
+    assert_eq!(
+        creator_balances,
+        coins(INITIAL_BALANCE + CREATION_FEE, NATIVE_DENOM)
+    );
+
     let num_tokens = 4;
     let (minter_addr, _config) = setup_minter_contract(&mut router, &creator, num_tokens).unwrap();
     // set to genesis mint start time
     setup_block_time(&mut router, GENESIS_MINT_START_TIME).unwrap();
 
+    // initial + creation fee - creation_fee = 2,000 + 1,000 - 1,000 = 2,000 tokens
```

Comment:
this comment will save us a lot of future frustrations

Reply:
It definitely gave me a lot of frustration today!